org.desmume.DeSmuME.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "org.desmume.DeSmuME",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "desmume",
+    "rename-desktop-file": "desmume.desktop",
+    "finish-args": [
+        "--filesystem=xdg-config/desmume:create",
```

Comment:
Just use $XDG_DATA_DIRS upstream and remove this permission

Reply:
Why?  Doesn’t this bind-mount `~/.config/desmume` into the sandbox, thus letting the user configure it from the documented directory in all cases?

Reply:
Users can still configure it from `.var/app/$app-id/config`.

Reply:
This is much less user-friendly though. :/

If possible I’d like to keep `~/.config/desmume` as the user configuration directory, to avoid having to modify all of the documentation everywhere.

That’s also what [I did in poezio](https://github.com/flathub/io.poez.Poezio/blob/master/io.poez.Poezio.json#L7) (where it is arguably even more important, since users are *expected* to modify these files manually).

Reply:
We made an exception for poezio as it's terminal app. Graphical apps should respect XDG_CONFIG_HOME.

Reply:
Alright, I’ve removed this line.